scaffolding-go18/plan.sh

```diff
@@ -0,0 +1,40 @@
+pkg_name=scaffolding-go18
```

Comment:
Following the naming convetion of plans themself, I think we want to actually call this `scaffolding-go`